partiql-parser/src/preprocessor.rs

```diff
@@ -18,11 +18,11 @@ pub(crate) enum FnExprArgMatch<'a> {
     /// Any 1 [`Token`] that is not function punctuation (i.e., '(', ')', ',') and not a keyword.
     ///
     /// Generally this will be followed by a [`AnyZeroOrMore`] match, in order to match 1 or more [`Token`]s
```

Comment:
```suggestion
    /// Generally this will be followed by a [`AnyZeroOrMore`] match, in order to match 1 or more [`Token`]s
    /// `bool` tuple value denotes if keyword is allowed to be considered as match.
```